diagnostic.md

```diff
@@ -2,37 +2,44 @@
 
 What are three advantages of using Ember?
 
-```sh
-# your answer here
+```
+Ember is uniform across websites, meaning that one looks similar to another and is easy
+to look at and understand, even if you didn't write the code.  It also has a big following on the web,
+great docs and great support by the people that wrote it.
```

Comment:
Those are great reasons to use it! Some more technical reasons include:
Front end routing
Easily display dynamic content using two-way binding
No more AJAX - Ember takes care of making API calls for you.

1/1